doc/source/whatsnew/v1.3.1.rst

```diff
@@ -16,6 +16,7 @@ Fixed regressions
 ~~~~~~~~~~~~~~~~~
 - Pandas could not be built on PyPy (:issue:`42355`)
 - :class:`DataFrame` constructed with with an older version of pandas could not be unpickled (:issue:`42345`)
+- Performance regression in constructing a :class:`DataFrame` from a dictionary of dictionaries (:issue:`42338`)
```

Comment:
should have been 42248.

Reply:
thanks, will add this to my next "collected cleanups" branch

Reply:
would need to backport, but no worries if added with other changes since the release readiness script will show that the release notes aren't in sync and I will sync directly to 1.3.x.

Reply:
done in #42516